task/testmap.py

```diff
@@ -27,9 +27,9 @@
     'cockpit-project/cockpit': {
         'master': [
             'fedora-31/container-bastion',
-            'fedora-31/selenium-firefox',
-            'fedora-31/selenium-chrome',
-            'fedora-31/selenium-edge',
+            'fedora-32/selenium-firefox',
+            'fedora-32/selenium-chrome',
+            'fedora-32/selenium-edge',
```

Comment:
After #1195 this now needs to be adjusted to remove the contexts from `_manual`. Technically not a blocker though, that cleanup could be a follow-up.

Reply:
right, but wanted to do it in this PR. Just pushed that and now spinning the tests.